src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -436,6 +436,22 @@ public void ExtractSwitchParametersTest()
             doubleQuotesRemovedFromArg.ShouldBe(6);
         }
 
+        [Fact]
+        public void GetLengthOfSwitchIndicatorTest()
+        {
+            var commandLineSwitchWithSlash = "/Switch";
```

Comment:
It might be good to have a test validating that building with `--help` works.

Reply:
+1

This might be overkill but: Consider a case for `-help`, `/help`, and `--help` and verifying part of the output is expected. Potentially in the `help` test just below this test.

Reply:
@BenVillalobos Can you elaborate on:
> verifying part of the output is expected

Thing is, that `MSBuildApp.Execute()` method only returns the `ExitType`.
How to get the string returned from command line execution?

Or perhaps checking whether each call (command with '*--help*', '*-h*, '*/h*') returns `ExitType.Success` will be enough?

Reply:
Verifying that it succeeded would be enough for me. I think it's probably more complicated than it's worth to look at the output, but you might be able to capture it with a logger you can pass in alongside the --help switch.